src/Microsoft.AspNetCore.Mvc.TagHelpers/PartialTagHelper.cs

```diff
@@ -74,6 +75,12 @@ public object Model
             }
         }
 
+        /// <summary>
+        /// Indicates if this partial view must be rendered
+        /// </summary>
+        [HtmlAttributeName(RequiredAttributeName)]
+        public bool Required { get; set; } = true;
```

Comment:
cc @NTaylorMullen 

Sort of the same argument as https://github.com/aspnet/Mvc/pull/7904#discussion_r196883543. Should this be inverted and named `Optional`. This would let you do:

`<partial name="SomeView" optional />` as opposed to `<partial name="SomeView" required="false" />`